kbs/src/token/mod.rs

```diff
@@ -22,25 +23,33 @@ pub trait AttestationTokenVerifier {
 pub enum AttestationTokenVerifierType {
     #[default]
     CoCo,
+    Jwk,
 }
 
 #[derive(Deserialize, Debug, Clone)]
 pub struct AttestationTokenVerifierConfig {
     #[serde(default)]
     pub attestation_token_type: AttestationTokenVerifierType,
 
-    // Trusted Certificates file (PEM format) path to verify Attestation Token Signature.
+    // Trusted Certificates file (PEM format) path (for "CoCo") or a valid Url
+    // (file:// and https:// schemes accepted) pointing to a local JWKSet file
+    // or to an OpenID configuration url giving a pointer to JWKSet certificates
+    // (for "Jwk") to verify Attestation Token Signature.
```

Comment:
Let's use `///` instead of `//` to work as inline documents.

Reply:
changed to `///`